Tools/Scripts/libraries/webkitscmpy/webkitscmpy/mocks/remote/git_hub.py

```diff
@@ -289,9 +290,66 @@ def _parents_of_request(self, url, ref):
             ), url=url
         )
 
-    def request(self, method, url, data=None, params=None, auth=None, json=None, **kwargs):
-        from datetime import datetime, timedelta
+    # FIXME: Not a very flexible mock of GitHub's GraphQL API, only supports pull-request querying
+    def graphql(self, url, auth=None, json=None):
+        query = (json or {}).get('query')
+        if not query:
+            return mocks.Response.create404(url)
+
+        qline = query.splitlines()[1]
+        pr_search = re.match(r'\s*search\(query:\s+"(?P<query>.+)",\s+type:\s+ISSUE,\s+last:\s+(?P<last>\d+)\)\s*\{', qline)
+        if pr_search:
```

Comment:
Maybe
```py
if not pr_search:
    return mocks.Response.create404(url)
```
to reduce the amount of indentation

Reply:
I want to keep it like this because if folks want to add more queries to this function (which, regrettably, seems quite likely), they'll just end up re-adding the indentation.